middleman_protocol/setup.py

```diff
@@ -1,13 +1,22 @@
 #!/usr/bin/env python3
-
+from os import path
+from pathlib import Path
 from setuptools import setup
 
-from version import get_version
+
+def generate_version() -> str:
+    here = path.abspath(path.dirname(__file__))
+    version_file = Path(path.join(here, "RELEASE-VERSION"))
+    if version_file.is_file():
+        return open(version_file, "r").read()
+    else:
+        from version import get_version
+        return get_version(prefix='v')
 
 
 setup(
     name='Middleman-Protocol',
-    version=get_version(prefix='v'),
+    version=generate_version(),
```

Comment:
This pull request changes only `middleman_protocol`. `signing_service` does this too so it also needs a fix (unless you're just planning to do it in a separate pull request).

Reply:
I will do it in this pull request. Done